framework/filters/AccessRule.php

```diff
@@ -28,14 +28,14 @@ class AccessRule extends Component
     /**
      * @var bool whether this is an 'allow' rule or 'deny' rule.
      */
-    public $allow;
+    public $allow = true;
```

Comment:
Shouldn't it be `false`? With `true` we are changing the default here.

Reply:
Doh! Yes. Fixed.